src/main/resources/service_config.json

```diff
@@ -273,5 +273,27 @@
         "READ",
         "UPDATE"
       ]
-    }]
+    },
+      {
+        "id": "nfdiv_case_api",
```

Comment:
@marvincudjoe - add the service names nfdiv_case_api and divorce_frontend in CASE_DOCUMENT_S2S_AUTHORISED_SERVICES for non-prod env.